bin/install-local-tests.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -e
+
+# Check if TMPDIR is set; if not, default to /tmp
+TMPDIR="/tmp"
+
+# Initialize a variable to hold the 'true' flag for skipping DB creation
+SKIP_DB=""
+
+# Check if '--no-db' is passed as an argument
+if [[ "$1" == "--no-db" ]]; then
+  SKIP_DB="true"
+fi
+
+# Run install-wp-tests.sh
+echo "Installing local tests into ${TMPDIR}"
+bash "$(dirname "$0")/install-wp-tests.sh" wordpress_test root '' 127.0.0.1 latest $SKIP_DB
```

Comment:
should these have the option to override?

Reply:
not sure what you mean. the point of the `$SKIP_DB` parameter is to optionally not install the database depending on whether you need it or not.

Reply:
i.e. on my local mariadb instance it got grumpy at me about using root/'', so for most local tests I have db creds configured with env variables for the given project with bashrc or .env. If this is a helper script for a human setting up testing, as opposed to CI, `wordpress_test root '' 127.0.0.1` might all want to be parameterized with defaults.

Reply:
oh, you mean the db password, yeah, i could see that being parameterized.

Reply:
I've made everything parameterized (or, ChatGPT did). I was mostly using this from the `composer test:install` and `composer test:install:withdb` scripts but a human can now run it manually with anything passed in as a flag.